_includes/exhibits.js

```diff
@@ -0,0 +1,1929 @@
+    });
+
+    $('#edit_description_modal form').submit(this, function(e){
+      const THIS = e.data;
+      const formData = parseForm(e.target);
+      $(this).closest('.modal').modal('hide');
+
+      // Get description from form
+      THIS.d = encode(formData.d);
+      $('#copy_link_modal').modal('show');
+
+      const root = THIS.location('host') + THIS.location('pathname');
+      const hash = THIS.makeHash(THIS.hashable.tag);
+      const link = document.getElementById('copy_link');
+      link.value = root + hash;
+
+      return false;
+    });
+
+    this.viewer.addHandler('canvas-enter', function(e) {
+      const THIS = e.userData;
+      THIS.faster();
+    }, this);
+
+    this.viewer.addHandler('canvas-drag', function(e) {
+      const THIS = e.userData;
+      const overlay = $('#' + THIS.currentOverlay);
+      const position = THIS.normalize(e.position);
+      if (THIS.drawing == 1) {
+        THIS.drawing = 2;
+        e.preventDefaultAction = true;
+        THIS.drawLowerBounds(position);
+      }
+      else if (THIS.drawing == 2) {
+        e.preventDefaultAction = true;
+        THIS.drawUpperBounds(position);
+      }
+    }, this);
+
+    this.viewer.addHandler('canvas-drag-end', function(e) {
+      const THIS = e.userData;
+      const position = THIS.normalize(e.position);
+      if (THIS.drawing == 2) {
+        e.preventDefaultAction = true;
+        THIS.finishDrawing(position);
+        THIS.pushState();
+      }
+    }, this);
+
+    this.viewer.addHandler('canvas-click', function(e) {
+      const THIS = e.userData;
+      const overlay = $('#' + THIS.currentOverlay);
+      const position = THIS.normalize(e.position);
+      if (THIS.drawing == 1) {
+        THIS.drawing = 2;
+        e.preventDefaultAction = true;
+        THIS.drawLowerBounds(position);
+      }
+      else if (THIS.drawing == 2) {
+        e.preventDefaultAction = true;
+        THIS.finishDrawing(position);
+        THIS.pushState();
+      }
+    }, this);
+
+    $(this.viewer.element).mousemove(this, function(e) {
+      const THIS = e.data;
+      THIS.mouseXY = e;
+      if (THIS.drawing == 2) {
+        THIS.drawUpperBounds(THIS.mouseXY);
+      }
+    });
+
+    this.viewer.addHandler('animation', function(e) {
+      const THIS = e.userData;
+      const scale = THIS.viewer.viewport.getZoom();
+      const pan = THIS.viewer.viewport.getCenter();
+      THIS.v = [
+        round4(scale),
+        round4(pan.x),
+        round4(pan.y)
+      ];
+      THIS.newView(false); 
+    }, this);
+
+    this.viewer.addHandler('animation-finish', function(e) {
+      const THIS = e.userData;
+      const scale = THIS.viewer.viewport.getZoom();
+      const pan = THIS.viewer.viewport.getCenter();
+      THIS.v = [
+        round4(scale),
+        round4(pan.x),
+        round4(pan.y)
+      ];
+      THIS.pushState();
+      THIS.faster();
+    }, this);
+
+    // Display viewer
+    this.finishAnimation();
+    this.viewer.setVisible(true);
```

Comment:
We need these two lines in order to avoid a visible camera transition when opening the application.